receiver/valkeyreceiver/README.md

```diff
@@ -0,0 +1,59 @@
+# Valkey Receiver
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics   |
+| Distributions | [contrib] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fvalkey%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Areceiver%2Fvalkey) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fvalkey%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Areceiver%2Fvalkey) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@dmitryax](https://www.github.com/dmitryax), [@rogercoll](https://www.github.com/rogercoll) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+The Valkey receiver is designed to retrieve Valkey INFO data from a single
+Valkey instance, build metrics from that data, and send them to the next consumer at a
+configurable interval.
+
+## Details
+
+The Valkey INFO command returns information and statistics about a Valkey
+server (see [https://valkey.io/commands/info/](https://valkey.io/commands/info/) for
+details). The Valkey receiver extracts values from the result and converts them to open
+telemetry metrics. Details about the metrics produced by the Valkey receiver
+can be found by browsing [the metadata file](./metadata.yaml).
+
+## Configuration
+
+> :information_source: This receiver is in beta and configuration fields are subject to change.
```

Comment:
I think this sentence is redundant, since the status of this component is in development, no one should assume that there will be no breaking changes to configuration.

Reply:
Good point, removed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37005/commits/018a5e481fba9a318dfe275f08b2f7b311be5976